scripts/lang/fr/template_handlers.py

```diff
@@ -0,0 +1,350 @@
+) -> str:
+    phrase = parts.pop(0)
+    if data["tr"]:
+        phrase += f", {italic(data['tr'])}"
+    if data["sens"]:
+        phrase += f" («&nbsp;{data['sens']}&nbsp;»)"
+    return phrase
+
+
+def render_lien_rouge(
+    tpl: str, parts: List[str], data: Dict[str, str], word: str, locale: str
+) -> str:
+    res = ""
+    if data["texte"]:
+        res = italic(data["texte"])
+    elif data["fr"]:
+        res = italic(data["fr"])
+    elif parts:
+        res = italic(parts[0])
+    elif data["trad"]:
+        res = italic(data["trad"])
+    return res
+
+
+def render_polytonique(
+    tpl: str, parts: List[str], data: Dict[str, str], word: str, locale: str
+) -> str:
+    phrase = parts.pop(0)
+    if data["tr"] or parts:
+        phrase += f", {italic(data['tr'] or parts.pop(0))}"
+    if data["sens"] or parts:
+        phrase += f" («&nbsp;{data['sens'] or parts.pop(0)}&nbsp;»)"
+    return phrase
+
+
+def render_siecle(
+    tpl: str, parts: List[str], data: Dict[str, str], word: str, locale: str
+) -> str:
+    parts = [part for part in parts if part.strip() and part not in ("lang=fr", "?")]
+    phrase = century(parts, "siècle") if parts else "Siècle à préciser"
+    return term(phrase)
+
+
+def render_suisse(
+    tpl: str, parts: List[str], data: Dict[str, str], word: str, locale: str
+) -> str:
+    if data["précision"]:
+        return term(f"Suisse : {data['précision']}")
+    else:
+        return term("Suisse")
+
+
+template_mapping = {
+    "acronyme": render_acronyme,
+    "argot": render_argot,
+    "agglutination": render_agglutination,
+    "déglutination": render_agglutination,
+    "dénominal": render_agglutination,
+    "déverbal": render_agglutination,
+    "déverbal sans suffixe": render_agglutination,
+    "parataxe": render_agglutination,
+    "reverlanisation": render_agglutination,
+    "syncope": render_agglutination,
+    "univerbation": render_agglutination,
+    "cit_réf": render_cit_ref,
+    "cit réf": render_cit_ref,
+    "composé de": render_compose_de,
+    "composé_de": render_compose_de,
+    "date": render_date,
+    "équiv-pour": render_equiv_pour,
+    "étyl": render_etyl,
+    "étylp": render_etyl,
+    "calque": render_etyl,
+    "la-verb": render_la_verb,
+    "supplétion": render_suppletion,
+    "zh-lien": render_zh_lien,
+    "forme reconstruite": render_recons,
+    "recons": render_recons,
+    "lien": render_lien,
+    "l": render_lien,
+    "LienRouge": render_lien_rouge,
+    "polytonique": render_polytonique,
+    "Polytonique": render_polytonique,
+    "siècle": render_siecle,
+    "Suisse": render_suisse,
+}
+
+
+def lookup_template(tpl: str) -> bool:
+    return tpl in template_mapping
+
+
+def render_template(template: Tuple[str, ...], locale: str, word: str = "") -> str:
+    tpl, *parts = template
+    data = extract_keywords_from(parts)
+    return template_mapping[tpl](tpl, parts, data, word, locale)
```

Comment:
We could go witout the `tpl` argument. Same for `locale` as it will not change.

Reply:
tpl is needed I'm afraid. Locale can go. Thanks

Reply:
Ah yes, because several templates are using the same function. That makes sense.